Cython/Compiler/FusedNode.py

```diff
@@ -711,45 +758,63 @@ def __pyx_fused_cpdef(signatures, args, kwargs, defaults, *, _fused_sigindex=(No
             self._buffer_declarations(pyx_code, decl_code, all_buffer_types, pythran_types)
             env.use_utility_code(Code.UtilityCode.load_cached("Import", "ImportExport.c"))
             env.use_utility_code(Code.UtilityCode.load_cached("ImportNumPyArray", "ImportExport.c"))
-
+        
+        context.update(
+            global_sigindex_name = "'__cython__fusedcpdef__sigindices'",
+            func_sigindex_keybase = self._get_unique_identifier_base()
+        )
+        
+        self._fused_sigindex_key(pyx_code)
+        
+        pyx_code.put_chunk(
+            u"""
+                globs = globals()
+                if {{global_sigindex_name}} not in <dict>globs:
+                    globs[{{global_sigindex_name}}] = {}
```

Comment:
This is really a hack. Doesn't a `global` variable work here?

Reply:
A `global` variable works here, but I wasn't sure I'd ever used them before, so I decided to minimize ~~variables~~ moving parts at first. :-)